curriculum/challenges/english/blocks/workshop-email-simulator/68bf0f09b7ec33be465c0c35.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 68bf0f09b7ec33be465c0c35
+title: Step 29
+challengeType: 20
+dashedName: step-29
+---
+
+# --description--
+
+At the end of the `__str__` method, return the email summary in this format:
+
+```py
+[status] From: [Sender name] | Subject: [Subject].
```

Comment:
```suggestion
[status] From: sender | Subject: subject.
```

it's confusing to know where you actually want the square brackets and where you don't. I don't want to field those kind of help request on the forum, so let's change it now please